crates/feldera-storage/src/file/mod.rs

```diff
@@ -210,7 +210,7 @@ mod test {
         after: &K,
         aux: A,
     ) where
-        S: StorageRead + StorageExecutor,
+        S: StorageRead + StorageControl + StorageExecutor,
```

Comment:
Should this sum of traits be another trait?

Reply:
We have a lot of traits here and I wonder whether ultimately they should be merged, or whether `StorageRead` and `StorageWrite` should each have `StorageControl` and `StorageExecutor` as supertraits.